pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -4193,6 +4193,98 @@ func TestChangefeedEnriched(t *testing.T) {
 	}
 }
 
+// TestChangefeedsParallelEnriched tests that multiple changefeeds can run
```

Comment:
Do you mind adding a comment explaining the motivation for this test? Is there some reason this wouldn't work (e.g. is there some kind of shared state for enriched changefeeds)?